features/checkout_details.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Check Out
+  If user click to checkout button when cart is empty
+  Return homepage
+
+Background: there are many products
+  Given there are five products in the application
+  
+@javascript
+Scenario: Return root page when cart hasnt item
+When I go to the homepage
+Then I should see the My Cart
+When I hover My Cart
+Then I dont see any item in cart and checkout button
+When I click on checkout button
+Then I return homepage
```

Comment:
Should use the word "should" in the Then description

`Then I should be redirected to homepage`